presto-main/src/main/java/com/facebook/presto/sql/gen/LambdaBytecodeGenerator.java

```diff
@@ -243,16 +302,44 @@ public BytecodeNode visitVariableReference(VariableReferenceExpression reference
         };
     }
 
-    static class LambdaExpressionField
+    static class CompiledLambdaMetadata
```

Comment:
@haozhun : Alternatively, we can just make sure we never use the shortened name (always use `compiledLambdaMetadata`